project/home/models.py

```diff
@@ -11,3 +15,47 @@ class HomePage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('intro', classname="full"),
     ]
+
+    def get_context(self,request):
+        context = super().get_context(request)
+        context['invite_friends'] = self.get_children().type(InviteFriendsPage)
+        print(context)
+        return context
+
+class FriendInvitation(models.Model):
+    email = models.EmailField(max_length=254,blank=True,default=None)
+
+
+class InviteFriendsPage(Page):
+
+    intro = RichTextField(blank=True,null=True,default="")
+    thankyou_page_title = models.CharField(
+        max_length=255, help_text="Title text to use for the 'thank you' page",blank=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('intro', classname="full"),
+        FieldPanel('thankyou_page_title'),
+    ]
+
```

Comment:
Adding `max_count = 1` will make it so only once instance of this page can be added, which makes sense since there is only one "Invite Friends" page.

```suggestion
max_count = 1
```